ServerCore/Pages/Teams/List.cshtml.cs

```diff
@@ -48,6 +50,11 @@ public async Task<IActionResult> OnGetAsync()
                 UserOnTeam = false;
             }
 
+            var applications = from TeamApplication application in _context.TeamApplications
+                                                           where application.Player == LoggedInUser && application.Team.Event == Event
+                                                           select application.Team;
+            AppliedTeam = await applications.FirstOrDefaultAsync();
```

Comment:
Any reason you did this on two lines? Usually we've been doing them in one. It's fine this way - I was just curious

Reply:
I copy/pasted this from elsewhere in the code. I think @morganbr uses this style. If I were to guess, I would say it's because when the query is written in this keyword-heavy syntax instead of a cascade of functions, you have to put the whole thing in parens to call FirstOrDefaultAsync and it'll look weird.